db/migrate/20160630150659_create_users.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.references :school, index: true, foreign_key: true
+      t.references :grove, index: true, foreign_key: true
+      t.string :name
+      t.string :email
+      t.string :password_digest
```

Comment:
@damwhit do you need this right now? I'm just thinking that devise may want a different password column... it looks like it may want something called `:encrypted_password`